services/core-web/src/components/noticeOfWork/NOWProgressActions.js

```diff
@@ -136,8 +140,8 @@ export class NOWProgressActions extends Component {
       content: modalConfig.NOW_PROGRESS_MODAL,
     });
 
-  openDraftPermitProgressModal = () =>
-    this.props.openModal({
+  openDraftPermitProgressModal = () => {
+    return this.props.openModal({
```

Comment:
Can you revert this back to an implicit return?

Reply:
done